apps/librelingo_yaml_loader/librelingo_yaml_loader/yaml_loader.py

```diff
@@ -22,10 +22,6 @@
 )
 from yaml import load
 
-try:
-    from yaml import CSafeLoader as SafeLoader
-except ImportError:
```

Comment:
how come this part didn't already work as a fallback for `CSafeLoader`?

Reply:
The first approach I tried is this one:
```
try:
    from yaml import CSafeLoader as SafeLoader
except ImportError:                                                               
    from yaml import SafeLoader                                                   
```
and it results in mypy error:
```
error: Incompatible import of "SafeLoader" (imported name has type "Type[SafeLoader]", local name has type "Type[CSafeLoader]")
```

Reply:
yes, just the mypy error.